domains/faizan.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "faizan.is-a.dev",
+  "subdomain": "faizan",
+  "owner": {
+    "email": "faizan.qureshi@upstrapp.com"
+  },
+  "records": {
```

It's `record` not `records`
```suggestion
  "record": {
```